app/lib/frontend/templates/views/page/topics_list.dart

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) 2023, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'dart:collection';
+
+import '../../../../shared/urls.dart' as urls;
+
+import '../../../dom/dom.dart' as d;
+
+d.Node renderTopicsList(Map<String, int> topics) {
+  return d.div(children: [
+    d.h1(text: 'Topics '),
+    topicsListNode(topics),
+  ]);
+}
+
+/// Renders the package list node of /topics page.
+d.Node topicsListNode(Map<String, int> topics) {
+  final sortedTopics = SplayTreeMap<String, int>.from(
```

Comment:
`topics.entries.toList()..sort((a, b) => -a.value.compareTo(b.value))` may be also good here, not sure which consumes less CPU

Reply:
Let's do this. It's also easier to read.